persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -304,7 +304,7 @@ MemberChangeContent.member_change_list AS memberChangeList,
 MemberChangeContent.member_change_type AS memberChangeType,
 ConversationNameChangedContent.conversation_name AS updateConversationName,
 Conversation.name AS conversationName,
-Mention.user_id AS mentionedUserId,
+(Mention.user_id IS NOT NULL) AS isMentioningSelfUser,
```

Comment:
is that enough to tell is that mentioning the self user?

Reply:
Ok, found it later. thanks

Reply:
Yup, because we are getting Mention only where Self.userId equals Mention.userId